community/detectors/apache_http_server_cve_2021_41773/src/test/java/com/google/tsunami/plugins/detectors/nacos/auth_bypass_lte140/ApacheHttpServerVulnDetectorTest.java

```diff
@@ -0,0 +1,128 @@
+package com.google.tsunami.plugins.detectors.nacos.auth_bypass_lte140;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.tsunami.common.data.NetworkEndpointUtils.forHostname;
+import static com.google.tsunami.common.data.NetworkEndpointUtils.forHostnameAndPort;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Guice;
+import com.google.inject.Inject;
+import com.google.tsunami.common.net.http.HttpClientModule;
+import com.google.tsunami.common.net.http.HttpStatus;
+import com.google.tsunami.common.time.testing.FakeUtcClock;
+import com.google.tsunami.common.time.testing.FakeUtcClockModule;
+import com.google.tsunami.plugins.detectors.rce.cve_2021_25646.ApacheHttpServerVulnDetector;
+import com.google.tsunami.proto.DetectionReportList;
+import com.google.tsunami.proto.NetworkEndpoint;
+import com.google.tsunami.proto.NetworkService;
+import com.google.tsunami.proto.TargetInfo;
+import com.google.tsunami.proto.TransportProtocol;
+import java.io.IOException;
+import java.time.Instant;
+import okhttp3.mockwebserver.Dispatcher;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ApacheHttpServerVulnDetectorTest {
+
+  private final FakeUtcClock fakeUtcClock =
+      FakeUtcClock.create().setNow(Instant.parse("2020-01-01T00:00:00.00Z"));
+
+  @Inject
+  private ApacheHttpServerVulnDetector detector;
+
+  private MockWebServer mockWebServer;
+
+  @Before
+  public void setUp() {
+    mockWebServer = new MockWebServer();
+  }
+
+  @After
+  public void tearDown() throws IOException {
+    mockWebServer.shutdown();
+  }
+
+  private void createInjector() {
+    Guice.createInjector(
+            new FakeUtcClockModule(fakeUtcClock),
+            new HttpClientModule.Builder().build())
+        .injectMembers(this);
+  }
+
+  @Test
+  public void testPathTraversalAndDisclosure()
```

Comment:
Please be consistent with the rest of this project and use `functionUnderTest_condition_outcome` as the naming schema for the tests.